doc/v2/dev/write_docs_en.rst

```diff
@@ -33,7 +32,7 @@ The tool uses Docker, please install it on your system. Please check Docker offi
     docker run -it -p 8000:8000 -v `pwd`:/var/content paddlepaddle/paddlepaddle.org:latest
 
 Note: PaddlePaddle.org will read the content repos specified in the -v (volume) flag of the docker run command
-Use a web browser and navigate to http://localhost:8000, click the buttons to compile the documentation
+Use a web browser and navigate to http://localhost:8000, click the buttons to compile the documentation.
```

Comment:
navigate to http://localhost:8000, click the -> navigate to **http://localhost:8000.** **Click** the

Reply:
Done